aria_plugin/executor.py

```diff
@@ -12,24 +12,69 @@
 #    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 #    * See the License for the specific language governing permissions and
 #    * limitations under the License.
-
+import os
+import pickle
+import subprocess
+import sys
+import tempfile
 from threading import Thread
 
 
-from cloudify import ctx
+# currently ARIA doesn't support providing a custom file to execute via the
+# process executor, thus process executor itself gets executed. This raises
+# an issue with packages missing __init__.py file at the root dir. in
```

Comment:
... the root dir. e.g. packages with multi top-level packages, such as ruaml. ...